chromium/README-Linux.md

```diff
@@ -0,0 +1,67 @@
+
+5. After successfully installing liboqs as per the above, navigate to `<CHROMIUM_ROOT>` and apply the `oqs-changes.patch` file provided here by running `git apply <PATH_TO_PATCH_FILE>`. Then, navigate to `third_party/boringssl`, and run `python src/util/generate_build_files.py gn`.
+
+Note: For this to succeed, you might have to install go if not already present on your machine, e.g., by running `sudo apt install golang-go`. If _any_ error occurs in this step, Chromium will build fine, just without support for quantum-safe crypto, i.e., only the final testing steps below will fail.
+
+6. Finally, navigate back to <CHROMIUM_ROOT> and follow the instructions [here](https://chromium.googlesource.com/chromium/src/+/master/docs/linux/build_instructions.md#Install-additional-build-dependencies) from the "Install additional build dependencies" section onwards to build Chromium. 
+
+Note: If you have already built another chromium source tree (version), you may have to execute `gclient sync --force` to ensure all dependencies are properly updated.
+
+Note: It is *strongly* advisable to set certain build options to obtain a size-and-performance optimized chromium variant, also saving on build time. Do this by executing `gn args out/Default` and adding the following variables to the configuration file opened in your editor:
+```
+# Set build arguments here. See `gn help buildargs`.
+is_debug = false
+symbol_level = 0
+enable_nacl = false
+blink_symbol_level=0
+```
+
+
+If the build completes successfully, i.e., the executable `chrome` has been created, one can verify that Chromium can perform a TLS 1.3 handshake using a post-quantum key exchange by executing these steps:
+
+0. Navigate to `<CHROMIUM_ROOT>`, and start Chromium by executing `./out/Default/chrome`
+1. Navigate again to the `<CHROMIUM_ROOT>/third_party/boringssl/src` folder, and build OQS-BoringSSL as a standalone project by running `mkdir build && cd build && cmake -GNinja ..`.
+2. Then, in the `build` directory, run `./tool/bssl server -accept 4433 -www -loop -curves <KEX>`, where `<KEX>` can be any key-exchange algorithm named [here](https://github.com/open-quantum-safe/boringssl#supported-algorithms) that is supported by default by Chromium. The [kDefaultGroups array](https://github.com/open-quantum-safe/boringssl/wiki/Implementation-Notes) lists all such algorithms\*.
+3. Load `https://localhost:4433` in Chromium.
+
+An alternative test consists of using the newly built Chromium to access the OQS test server at [https://test.openquantumsafe.org](https://test.openquantumsafe.org) and clicking on any of the algorithm combinations [supported by Chromium](https://github.com/open-quantum-safe/boringssl/blob/master/ssl/t1_lib.cc#L375), e.g., `p256_kyber512`).
```

Comment:
When looking up this link, it returns "404". The `boringssl` update PR (https://github.com/open-quantum-safe/boringssl/pull/103) also doesn't seem to reference/contain this file.

Reply:
> Do you want me to remove README-Linux.md ? Or we can change the name of README-Linux.md to README-Linux-OUTDATED.md .

No -- if it is meant to contain _current_ instructions how to build for Linux (and you write in the new "README.md" that everything has been tested under Linux), it should be updated with current instructions (containing correct references).

Reply:
> I think we can merge README-Linux and README-Windows into a single file in the future

That would probably be best.

> I also modified the git command so it will fetch a specific boringssl commit

This seems very risky to me: Any future updates to oqs-boringssl will then not be picked up any more. But then again, you agreed to look after that subproject, so I trust you'll update these explicit commit references also here whenever you do changes there.